reports/metricas_anual_y_por_partido.html

```diff
@@ -14,16 +14,15 @@
     </head>
     <body>
 		<div class="container-lg">
-      <h1>Métricas de los cimarrones</h1>
+      <h1>Cimarrones de Sonora F.C.</h1>
       <h2>Métricas de referencia</h2>
             <p class="lead" style="font-size:1.7em">
-            En la figura 1 podemos observar la posesión en algunos partidos de los Cimarrones.
-            Tomamos solo seis partidos para hacer un muestra de esta métrica. Las barras
+            En la figura 1 podemos observar la posesión en algunos partidos de los Cimarrones. Las barras
             horizontales azules representa la posesión de los Cimarrones y las barras rojas a sus
-            rivales.
+            rivales. Tomamos solo seis partidos para hacer un muestra de esta métrica.
```

Comment:
Aquí hubo un error de dedo:
```suggestion
            rivales. Tomamos solo seis partidos para hacer una muestra de esta métrica.
```